gamemodes/terrortown/gamemode/client/cl_main.lua

```diff
@@ -113,7 +121,14 @@ function GM:Initialize()
 	self.roundCount = 0
 
 	-- load addon language files
-	LANG.SetupFiles("lang/", true)
+	fileloader.LoadFolder("lang/", true, CLIENT, function(path)
+		ErrorNoHalt("[DEPRECATION WARNING]: Loaded language file from 'lang/', this folder is deprecated. Please switch to `terrortown/lang/`\n")
```

Comment:
I wouldn't make this an Error. This will popup for every user using an add-on that handles language files in the old way. Better making it a `MsgN` as well.